specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/kustoPool.json

```diff
@@ -1091,7 +1091,7 @@
           "Databases"
         ],
         "description": "Returns the list of databases of the given Kusto pool.",
-        "operationId": "Databases_ListByKustoPool",
+        "operationId": "KustoDatabases_ListByKustoPool",
```

Comment:
Suggest renaming to '**KustoPool**Databases_ListByKustoPool', to differentiate between Kusto Pool sub-resources and original Kusto resources.
[Same comment applies to other changes]